pkg/actions/project.go

```diff
@@ -135,3 +139,106 @@ func ProjectRemoveConnection(c *cli.Context) {
 	fmt.Println(string(response))
 	os.Exit(0)
 }
+
+// ProjectList : Print the list of projects to the terminal
+func ProjectList(c *cli.Context) {
+	conID := strings.TrimSpace(strings.ToLower(c.String("conid")))
+
+	conInfo, conInfoErr := connections.GetConnectionByID(conID)
+	if conInfoErr != nil {
+		fmt.Println(conInfoErr.Err.Error())
+		os.Exit(1)
+	}
+
+	conURL, conErr := config.PFEOriginFromConnection(conInfo)
+	if conErr != nil {
+		fmt.Println(conErr.Err.Error())
+		os.Exit(1)
+	}
+
+	projects, err := project.GetAll(http.DefaultClient, conInfo, conURL)
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
Might be worth checking for JSON global flag and outputting utils.PrettyPrintJSON(conErr) you can't use HandleProjectError here because project.GetAll returns a go err.  Could project.GetAll return a projectErr ?